src/less/select/select.less

```diff
@@ -35,7 +35,7 @@ span.select {
 }
 
 .select--large select {
-    font-size: @font-size-18;
+    font-size: var(--font-size-18);
```

Comment:
Directly using `--font-size-18` instead of a variant here

Reply:
Would like to create a new variant for consistency `--font-size-medium-2` or `--font-size-large-0` but there will be a mismatch with DS. Should I go this approach by talking to DS?

Reply:
Created a new issue to fix this later #2175